srtcore/api.h

```diff
@@ -264,17 +261,9 @@ friend class CRendezvousQueue;
 };
 
 // Debug support
-inline std::string SockaddrToString(const sockaddr* sadr)
+inline std::string SockaddrToString(const sockaddr_any& sadr)
 {
-    void* addr =
-        sadr->sa_family == AF_INET ?
-        (void*)&((sockaddr_in*)sadr)->sin_addr
-        : sadr->sa_family == AF_INET6 ?
-        (void*)&((sockaddr_in6*)sadr)->sin6_addr
-        : 0;
-    // (cast to (void*) is required because otherwise the 2-3 arguments
-    // of ?: operator would have different types, which isn't allowed in C++.
-    if ( !addr )
+    if (sadr.family() != AF_INET && sadr.family() != AF_INET6)
```

Comment:
Not exactly because when you pass sockaddr initialized as `0.0.0.0:0`, this would be satisfied condition for `empty()`, whereas it should be exactly displayed as `0.0.0.0:0`, not `unknown:0`.